src/model/contact-information/fetched-repository.ts

```diff
@@ -0,0 +1,19 @@
+import axios from "axios";
+import config from "src/config";
+
+import { ContactInformationDto, ContactInformationRepository, ContributorId } from "./repository";
+
+export class FetchedContactInformationRepository implements ContactInformationRepository {
+  public async findByContributorId(contributorId: ContributorId): Promise<ContactInformationDto> {
+    const response = await axios.get<ContactInformationDto>(`${config.DATA_API_HOSTNAME}/contributors`, {
```

Comment:
I was about to tell you, but you implemented this too fast: the endpoint will finally be `GET /contributors/{contributorId}/contact-information`

BTW, the endpoint to save (create or update) contact-information will be `PUT /contributors/{contributorId}/contact-information`